cips/cip-14.md

```diff
@@ -213,7 +213,18 @@ This proposal is backwards-incompatible because it is state-machine breaking. Th
 
 ## Test Cases
 
-TBA
+The following test cases are available in the [ibc-go e2e repository.](https://github.com/cosmos/ibc-go/tree/main/e2e/tests/interchain_accounts)
+
+- Registration of an interchain account
+- Transfer funds from interchain account to a different account on the same chain using an unordered or ordered channel
+- A failed transfer of funds from interchain account to a different account on the same chain due to insufficient funds
+- Transfer funds from interchain account to a different account on the same chain after an ordered channel closure
+- A transfer of funds from an interchain account to a different account on the same chain using a gov controller
```

Comment:
What is a gov controller? I don't think it was mentioned in this CIP or in https://github.com/cosmos/ibc/blob/main/spec/app/ics-027-interchain-accounts/README.md

Reply:
clarified that its from an x/gov sdk module based controller to send packets to the interchain account